### DISCUSSION TITLE
Support for HTTP HEAD request

### DISCUSSION THREAD

**Discussion by schmallaria** [grafana/grafana-infinity-datasource#711] [Ideas]
Hi,

i'm trying to peek some messages from an Azure Service Bus using the [this method](https://learn.microsoft.com/en-us/rest/api/servicebus/peek-lock-message-non-destructive-read). 

To get messages from an Azure Service Bus the client should send a POST request to the Azure API. 

The response consists of two parts:

- message properties are written to the HTTP response header
- the message itself is written to the HTTP body

Sample request
`POST https://your-namespace.servicebus.windows.net/HttpClientSampleQueue/messages/head?timeout=60`

Sample response
```
HTTP/1.1 201 Created  
Transfer-Encoding: chunked  
Content-Type: application/atom+xml;type=entry;charset=utf-8  
Location: https://your-namespace.servicebus.windows.net/httpclientsamplequeue/messages/2/7da9cfd5-40d5-4bb1-8d64-ec5a52e1c547
Server: Microsoft-HTTPAPI/2.0  
BrokerProperties: {"DeliveryCount":1,"EnqueuedSequenceNumber":0,"EnqueuedTimeUtc":"Wed, 02 Jul 2014 01:32:27 GMT","Label":"M1","LockToken":"7da9cfd5-40d5-4bb1-8d64-ec5a52e1c547","LockedUntilUtc":"Wed, 02 Jul 2014 01:33:27 GMT","MessageId":"31907572164743c38741631acd554d6f","SequenceNumber":2,"State":"Active","TimeToLive":10}  
Priority: "High"  
Customer: "12345,ABC"  
Date: Wed, 02 Jul 2014 01:32:27 GMT  
    
12  
This is a message.  
0
```

**Features requested:**

- Support to fetch values from the HTTP response header

Do you think it's possible to implement such a feature?

**Comment by naveenkalithkar** [comment]
Hi @yesoreyeram ,

Infinity plugin is awesome and very helpful thank you for awesome work.
Is it possible to add this feature so that we can make use of the header data or is there any way to achieve it using Infinity plugin.

**Comment by yesoreyeram** [comment]
Hiya. There are no plans to add support for this due to potential security risks involved.
